src/views/virtualmachines/actions/components/DeleteVMModal/hooks/useDataVolumeConvertedVolumeNames.ts

```diff
@@ -21,14 +21,9 @@ const useDataVolumeConvertedVolumeNames: UseDataVolumeConvertedVolumeNames = (vm
     .filter((volume) => volume?.dataVolume)
     ?.map((volume) => volume?.dataVolume?.name);
 
-  const pvcVolumesNames = (vmVolumes || [])
-    .filter((volume) => volume?.persistentVolumeClaim)
-    ?.map((volume) => volume?.persistentVolumeClaim?.claimName);
   return {
     dvVolumesNames,
-    pvcVolumesNames: isDataVolumeGarbageCollector
-      ? pvcVolumesNames.concat(dvVolumesNames)
-      : pvcVolumesNames,
+    pvcVolumesNames: isDataVolumeGarbageCollector ? dvVolumesNames : [],
```

Comment:
Hmm, does it still make sense to call this "pvcVolumesNames"?

Reply:
I'll propose a new thing in order to fix that.